test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua

```diff
@@ -0,0 +1,144 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description: Check HMI status transitions of WEB_VIEW application
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) WebEngine App with WEB_VIEW HMI type is registered
+-- 3) WebEngine App is in some state
+--
+-- Steps:
+-- 1) WebEngine App is moving to another state by one of the events:
+--   App activation, App deactivation, Deactivation of HMI, User exit
+-- SDL does:
+--  - send OnHMIStatus notification with appropriate value of 'hmiLevel' parameter
+--
+-- Particular behavior and value depends on initial state and event, and described in 'testCases' table below
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Test Configuration ]]
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```

Comment:
This restriction is added to `commonWebEngine` so can be removed from the script.